build.sbt

```diff
@@ -1,27 +1,11 @@
-import sbtrelease.ReleaseStateTransformations._
-
-
-resolvers += Resolver.sonatypeRepo("releases")
-
-val bintrayReleaseProcess: Seq[ReleaseStep] = Seq(
-  releaseStepTask(test),
-  inquireVersions,
-  setReleaseVersion,
-  releaseStepTask(publish),
-  commitReleaseVersion,
-  setNextVersion,
-  commitNextVersion
-)
-
 val common = Seq(
   scalaVersion := "2.12.6",
-  organization := "com.ovoenergy",
+  organization := "com.ovoenergy.effect",
```

Comment:
👋 shouldn’t the organization be just “com.ovoenergy”?

Reply:
@tomverran, as per Erica's comment above, shouldn't we change the org to `com.ovoenergy`?

Reply:
oh I had some idea I'd made a code comment about this - I didn't want to take `com.ovoenergy.logging`, `com.ovoenergy.metrics` etc at the top level and since all the modules are under `com.ovoenergy.effect` this seemed sensible

Reply:
oh I see, yeah that sounds like a good idea.